samples/maven-publish/README.md

```diff
@@ -0,0 +1,5 @@
+maven-publish
+==========
+
+Demonstrates how to apply and configure the `maven-publish` plugin to publish a kotlin library with accompanying
+sources.
```

Comment:
:x: Should add some guidelines on how to run the sample (`publish` task) and where the sample repository is.

Reply:
Done.